omero/sysadmins/limitations.txt

```diff
@@ -143,6 +143,15 @@ clients will use the extrema of the pixel type range by default. Users can
 adjust the minima/maxima via the rendering settings. Recalculating minima and
 maxima pixel values after import is currently not supported.
 
+Flex data with Acapella results in OMERO.tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are currently issues with the configuration required to access the raw
+data in combination with the analysis results via
+:doc:`OMERO.tables </developers/analysis>`. You may not be able to view the
+ROIs overlayed on the associated image by selecting the results they relate to
```

Comment:
@joshmoore No, not explicitly, I think we should go for what you suggest and ignore the ROIs. I was just trying to explain the feature but I did not know what is broken and what is not. Yes, agree, this is too much divined here. Probably leave out.